editor/src/components/canvas/controls/reorder-slider-control.tsx

```diff
@@ -46,6 +46,10 @@ export const ReorderSliderControl = controlForStrategyMemoized(
         store.editor.canvas.interactionSession.activeControl.type === 'REORDER_SLIDER',
       'ReorderSliderControl isDragging',
     )
+    const isSelectedElementHovered = useEditorState((store) => {
+      const { hoveredViews, selectedViews } = store.editor
+      return hoveredViews.includes(selectedViews[0])
```

Comment:
Should this check all of the possible values of `selectedViews`? I don't _think_ it should, but I wonder if that's more coincidental to how the related strategies are implemented.